addon/lib/toolbar-button.js

```diff
@@ -8,17 +8,20 @@ const {Panel} = require('sdk/panel');
 const querystring = require('sdk/querystring');
 const store = require('sdk/simple-storage').storage;
 const tabs = require('sdk/tabs');
+const self = require('sdk/self');
 
 const Mustache = require('mustache');
 const templates = require('./templates');
 Mustache.parse(templates.installed);
 Mustache.parse(templates.experimentList);
 
+const xulcss = require('./xulcss');
+xulcss.addXULStylesheet(self.data.url('button.css'));
```

Comment:
Drive-by review: You probably want to remove the stylesheet on disable/uninstall, too.  It wouldn't _really_ hurt anything to leave it added, but it's nice to clean up after yourself.  😉